common/districts/!!_hfe_district.txt

```diff
@@ -80,9 +83,11 @@ hfe_district_memorial_scavenger = {
 		}
 	}
 	ai_resource_production = {
-		minerals = 1
 		trigger = {
 			always = yes
 		}
+		energy = 15
```

Comment:
I see no reason why ai should get such bonuses

Reply:
Balance. The AI is very, very stupid and it will consistently get deficits.

Reply:
Vanilla districts have no such cheats

Reply:
These aren't Vanilla rendition Districts. Bottom line.

Reply:
And?
I do not see why 4 districts in galaxy should have cheats for ai

Reply:
Memorial Worlds are unorthodox and have different dynamics, Districts, and more. In adjusting this, I factored in balance and have tested it. It works.

Reply:
Still i do not see why ai should get this cheats.

Reply:
You underestimate the stupidity of the AI.

Reply:
Ai have cheat resources from pdx.
Second, ai will not be saved from 15 energy from one district from one planet